kalite/settings/base.py

```diff
@@ -322,6 +323,9 @@
 
 ROOT_URLCONF = "kalite.distributed.urls"
 
+BACKUP_DIRPATH = os.path.join(expanduser("~"), 'ka-lite-backups')
```

Comment:
I like the current behaviour better. Some users might mindlessly delete the .kalite directory. Having a visible folder for backups makes it clear that 1. backups are being done and 2. it's stored in here.